### PR TITLE
version: 0.0.1

### PR DESCRIPTION
**Pull request by jcvenegas**
Fix version to be the same to all kata projects.

Fixes: #223

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1,2 @@
-0.1.0
+# This is the runtime version.
```
Comment (bergwolf) [inline VERSION]: Please no comment lines. Just the version. The Makefile relies on it.

Comment (jcvenegas) [reply VERSION]: @bergwolf  removed comments.

**Comment by bergwolf** [comment]
lgtm!

**Comment by gnawux** [comment]
lgtm

**Comment by WeiZhang555** [comment]
LGTM :+1:

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**